portfolio/src/main/webapp/gallery.js

```diff
@@ -0,0 +1,19 @@
+var imageIndex = 1;
+var totalimg = 7;
+function changeImage(x) {
+  imageIndex += x;
+  if(imageIndex > totalimg){
+    imageIndex = 1;
+  }
+  if(imageIndex <= 0){
+    imageIndex = totalimg;
+  }
+  const imgUrl = 'images/img' + imageIndex + '.jpg';
```

Comment:
This piece of code can be replaced by following:
```suggestion
var imageIndex = 0;
var totalimg = 7;
function changeImage(x) {
  imageIndex = (imageIndex + x + totalimg) % totalimg;
  const imgUrl = 'images/img' + (imageIndex + 1) + '.jpg';
```